buildpipeline/DotNet-CoreFx-Trusted-Linux-Crossbuild.json

```
@@ -59,6 +59,25 @@
         "Contents": "**\n.gitignore\n.editorconfig\n.gitattributes\n.gitmirrorall\n.git/**\n.git"
       }
     },
+    {
+      "environment": {},
+      "enabled": true,
+      "continueOnError": true,
+      "alwaysRun": false,
+      "displayName": "Delete files from $(PB_GitDirectory) again",
+      "timeoutInMinutes": 0,
+      "task": {
+        "id": "d9bafed4-0b18-4f58-968d-86655b4d2ce9",
+        "versionSpec": "1.*",
+        "definitionType": "task"
+      },
+      "inputs": {
+        "filename": "rm",
+        "arguments": "-rf $(PB_GitDirectory)",
```

To be safe (if the sources directory somehow has a space in it) I think we should quote this:

```suggestion
        "arguments": "-rf \"$(PB_GitDirectory)\"",
```